finagle-netty4/src/main/scala/com/twitter/finagle/netty4/ssl/SslConnectHandler.scala

```diff
@@ -7,6 +7,28 @@ import io.netty.util.concurrent.{Future => NettyFuture, GenericFutureListener}
 import java.net.SocketAddress
 import javax.net.ssl.SSLSession
 
+private[netty4] class SslServerConnectHandler(ssl: SslHandler) extends ChannelInboundHandlerAdapter { self =>
```

Comment:
Maybe put it in its own file? Also rename `SslConnectHandler` to `SslClientConectHandler`?